x/gamm/pool-models/stableswap/amm.go

```diff
@@ -263,7 +266,9 @@ func solveCFMMBinarySearchMulti(xReserve, yReserve, wSumSquares, yIn osmomath.Bi
 	}
 
 	maxIterations := 256
-	errTolerance := osmoutils.ErrTolerance{AdditiveTolerance: sdk.OneInt(), MultiplicativeTolerance: sdk.Dec{}}
+
+	// we use a geometric error tolerance that guarantees approximately 10^-8 precision on outputs
```

Comment:
```suggestion
	// we use a geometric error tolerance that guarantees approximately 10^-12 precision on outputs
```

Maybe move this to a constant in types?